README.md

```diff
@@ -1,2 +1,67 @@
-# RFCs
-RFCs for the Open 3D Foundation products
+[O3DE RFCs]: #O3DE-rfcs
```

Comment:
I think the page is missing the actual heading for this anchor. "O3DE RFCs" text does not display on the page.